op-batcher/batcher/channel_builder.go

```diff
@@ -197,14 +204,9 @@ func (c *ChannelBuilder) Blocks() []*types.Block {
 	return c.blocks
 }
 
-// Reset resets the internal state of the channel builder so that it can be
-// reused. Note that a new channel id is also generated by Reset.
-func (c *ChannelBuilder) Reset() error {
-	c.blocks = c.blocks[:0]
-	c.frames = c.frames[:0]
-	c.timeout = 0
-	c.fullErr = nil
-	return c.co.Reset()
+// LatestL1Origin returns the latest L1 block origin from all the L2 blocks that have been added to the channel
+func (c *ChannelBuilder) LatestL1Origin() eth.BlockID {
+	return c.latestL1Origin
 }
```

Comment:
I removed the `Reset()` method as it didn't appear to be used and keeping it would complicate the tracking of the latest l1 origin block.

Reply:
Thanks, and interesting what it was for 😅